dev-util/rebar/rebar-3.16.0.ebuild

```
@@ -0,0 +1,91 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1
+
+HEX_PACKAGES="
+	bbmustache-1.10.0
+	certifi-2.6.1
+	cf-0.3.1
+	cth_readable-1.5.1
+	erlware_commons-1.5.0
+	eunit_formatters-0.5.0
+	getopt-1.0.1
+	meck-0.8.13
+	providers-1.8.1
+	relx-4.4.0
+	ssl_verify_fun-1.1.6
+"
+
+hex_package_uris() {
+	local package
+	for package in "$@"; do
+		echo "https://repo.hex.pm/tarballs/$package.tar -> $package.hex.tar"
```

```suggestion
		echo "https://repo.hex.pm/tarballs/${package}.tar -> ${package}.hex.tar"
```
Let's add some bashism.